exchanges/bybit/bybit_websocket.go

```diff
@@ -0,0 +1,547 @@
+package bybit
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gorilla/websocket"
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/common/crypto"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/orderbook"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/stream"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/ticker"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/trade"
+	"github.com/thrasher-corp/gocryptotrader/log"
+)
+
+const (
+	bybitWSBaseURL      = "wss://stream.bybit.com/"
+	wsSpotPublicTopicV2 = "spot/quote/ws/v2"
+	wsSpotPrivate       = "spot/ws"
+	bybitWebsocketTimer = 30 * time.Second
```

Comment:
This needs to be 20 seconds

```
[WARN]  | WEBSOCKET | 01/08/2022 09:46:54 | Bybit websocket has been disconnected. Reason: websocket: close 1006 (abnormal closure): unexpected EOF
```
![image](https://user-images.githubusercontent.com/9391715/182051453-b6170fb9-1101-45c2-8bdb-a4f534925361.png)